src/containers/CookieContainer.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react'
+import CookieConsentBanner from '../components/CookieConsentBanner'
+
+const CookieContainer = () => {
```

Comment:
Do we actually need this container, since it's not connected to the redux store?